gendox-core-api/src/main/java/dev/ctrlspace/gendox/gendoxcoreapi/services/TrainingService.java

```diff
@@ -105,13 +111,16 @@ public Embedding runTrainingForSection(@NonNull DocumentInstanceSection section,
         EmbeddingResponse embeddingResponse = embeddingService.getEmbeddingForMessage(project.getProjectAgent(),
                 sectionValue,
                 project.getProjectAgent().getSemanticSearchModel());
-        Embedding embedding = embeddingService.upsertEmbeddingForText(embeddingResponse, projectId, null, section.getId(), project.getProjectAgent().getSemanticSearchModel().getId(), project.getOrganizationId());
+
+        String sectionSha256Hash = cryptographyUtils.calculateSHA256(section.getSectionValue());
```

Comment:
Are you sure that this is the ACTUAL text for what we got embedded? What about the Template transformation?

In the `DocumentInstanceSectionProcessor` the template is used.

If this service method, is NEVER used, we can delete it completely...